microsoft/testsuites/mshv/mshv_install.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+from typing import Any, Dict
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.testsuite import TestResult
+from lisa.tools import Cp, Ls, Reboot
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="mshv",
+    category="functional",
+    description="""
+    This test suite is to test VM working well after updating MSHV on VM
```

Comment:
Give full name of MSHV. The description will be in LISA document, it should be understandable for others.